### DISCUSSION TITLE
install erasure component， must to install datanode and metanode？

### DISCUSSION THREAD

**Discussion by lddlww** [cubefs/cubefs#3491] [Q&A] [answered]
i want to use erasure code in pro；when i install erasure component like clustermgr，access，proxy，blobnode，scheduler；i have to prepare datanode and metanode ？

**Comment by FourSpaces** [comment, marked as answer]
No need to prepare DataNode, but MetaNode and Master are required

**Reply by lddlww** [reply]
thanks
